src/public/map.js

```diff
@@ -221,22 +228,35 @@ function removeAllFeatures() {
   markers.clearLayers();
 }
 
-// The brighter the red, the more ML features the EA user uses.
-// 0 features is black circle marker
-// 3+ creates a bright red circle marker
-// f is a EA user
+// Color of map marker corresponds to number of features the user uses
+// black: 0 features
+// red: 1 feature
+// Green: 2 features
+// Yellow: 3+ features
 function getColor(user) {
   var numFeatures = 0;
   if (user.properties.features && user.properties.features.length) {
     numFeatures = user.properties.features.length;
-  } // 57 + 66(3) = 255
-  var red = 57 + 66 * numFeatures;
-  // Color doesn't display correctly if > 255
-  red = red > 255 ? 255 : red;
-  //toString(16) converts number to base 16 string ex. 10 -> a
-  var c = "#"+red.toString(16)+(50).toString(16)+(50).toString(16);
-
-  return c;
+  }
+
+  var ff = 'ff';
+  var red = '00', green = '00', blue = '00';
```

Comment:
I think this is a bit too clever -- it would take a little time for the next developer to figure out what you're doing, compared to something like:

    var color = '';
    if (numFeatures === 0) {
      color = '#000';
    } else if (numFeatures === 1) {
      color = '#f00';
    ...
    }
    return color;